src/test/test.js

```diff
@@ -45,7 +45,7 @@ describe('Registrar', () => {
       // fs.writeFileSync('src/test/contracts.json', JSON.stringify(compiled));
       // Use to speed up the testing process during development:
       const compiled = JSON.parse(fs.readFileSync('src/test/contracts.json').toString());
-      const deployer = compiled['DeployENS']; // eslint-disable-line
+      const deployer = compiled[':DeployENS']; // eslint-disable-line
```

Comment:
Solidity was updated to return compiled data in the format "filename:contractname". When being used without a filename, that part is empty.